### DISCUSSION TITLE
Web3:// Sounds 项目组队，永久记录每一个人的声音

### DISCUSSION THREAD

**Discussion by luffythink** [IntensiveCoLearning/Web3-URL#153] [Ideas and Project Teams]
- **暂定取名**： Web3:// Sounds  🤔
- **slogan** ：
  - 永久记录每一个人的声音。
  - 连接声音，探索未来世界计算机。
- **想法构思中**：
  - **主题**：聚焦于围绕Web3:// 协议构建「未来世界计算机」这一核心主题
  - **轻量**：这里更关注每一个人的声音，而不是 1 对多，或几对多的声音分享
  - **发声**：每一个人都可以基于未来世界计算机这一主题用嘴巴讲出自己的想法，可以是一个字、一句话、一段音乐等等都行，关键在于是你内心的声音
  - **智能合约记录**：Web3:// Talks 会将这些声音永久的记录在去中心化全链网络里，智能合约会按照时间、性别、音调、关键词等一些属性（前期可能属性很简单），将这些声音按照某种算法排序
  - **前端呈现**：前端会呈现一个不同声音的连接网状图，呈现一个有趣的声音图景
    - **访问**：在前端页面里可以点击声音图景上的声音进行聆听，每一次页面呈现的图景都不同，声音也不同
    - **互动**：如果你喜欢某个声音，可以长按互动（未来功能，涉及有趣的互动形式）
- **小愿景**：每一个人都可以发声，每一个人的声音都将被永久记住，联合能连接更多人的声音，一起对话未来世界计算机。

欢迎大家脑洞，
需要前端、solidity 合约设计小伙伴，
基于 Web3:// 协议实现提炼前期能完成的初步功能设想，完成 demo 及一些基础功能一起组队 Buidl 完成。
技术实现不知道难度系数高不高，先起个头试试，重在有趣，好玩。

**Comment by DerickIT** [comment]
听起来很有意思，我想参与下，一个后端兼合约选手

**Comment by luffythink** [comment]
画了一个草图：🤔 Web 山 之声
![web3sounds](https://github.com/user-attachments/assets/859dbe45-32d2-4fcb-8dfa-952b8b32c14f)

还差前端选手，有木有加入的呀。

**Reply by CHENFANGC** [reply]
hi~ 对项目感兴趣，还缺前端么~

**Reply by luffythink** [reply]
来一起，加微信 211951368 我拉群。
